tables_script.sql

```diff
@@ -1,28 +1,29 @@
 -- Automation system runs table
 CREATE TABLE IF NOT EXISTS `biosero_uat`.`automation_system_runs` (
   `id` INT NOT NULL AUTO_INCREMENT,
-  `automation_system` VARCHAR(255) NOT NULL,
-  `run_id` INT NOT NULL UNIQUE,
+  `automation_system_type` ENUM('biosero', 'beckman', 'sentinel') NOT NULL,
+  `automation_system_name` ENUM('CPA', 'CPB') NOT NULL,
+  `system_run_id` INT NOT NULL UNIQUE,
```

Comment:
Can we add comments to a create query? Might be worth adding some to know the differences between the `id` and `system_run_id`; especially for those who haven't been in the discussions and for me in one months time when I've forgotten everything!

Reply:
yes, adding